common/config.go

```diff
@@ -78,9 +67,6 @@ func LoadFromCommandLine(config *Config) (*Config, error) {
 
 	flagSet.StringVar(&config.ConfigFile, "conf-file", "", "File path of configuration file")
 
-	flagSet.StringVar(&config.PluginDir, "plugin-dir", config.PluginDir, "Docker plugin directory where driver socket is created")
-	flagSet.StringVar(&config.DriverName, "driver-name", config.DriverName, "Name of Infoblox IPAM driver")
```

Comment:
Keeping @johnbelamaric  in the loop...

If it cannot be supported by Docker, then the only thing we can do is to allow passing in additional IPAM configurations (e.g. network view and network container) as ipam-opt during networking creation.